.github/workflows/reject-staging-changes.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow fails in PRs with changes to the jupyterlab/staging directory
+# Won't start unless there is a modification in the staging directory
+
+name: 'Reject changes to staging'
+on:
+  pull_request:
+    paths:
+      - 'jupyterlab/staging/**'
+
+jobs:
+  staging-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Get modified files in the staging directory
+        id: modified-files-in-staging
+        uses: tj-actions/changed-files@v18.6
+        with:
+          # only checks for modified files in this directory
+          files: jupyterlab/staging
+          # ignores modifications to these files
+          files_ignore: |
+            jupyterlab/staging/yarn.js
```

Comment:
Looks like we could also add `.yarnrc` to the list, since it's kept here:

https://github.com/jupyterlab/jupyterlab/blob/5aa5f62678dabcc6bdff0d98b6368652ed7713ee/buildutils/src/update-core-mode.ts#L24-L25

Reply:
sure, just added it

Reply:
thanks!